FrostyModSupport/FrostyModExecutor.cs

```diff
@@ -253,6 +253,13 @@ private int HashBundle(BundleEntry bentry)
 
         private void ProcessModResources(IResourceContainer fmod)
         {
+            // Bundle whitelist may not contain the chunk bundle. This adds it to prevent issues
+            if (App.WhitelistedBundles.Count != 0 && !App.WhitelistedBundles.Contains(chunksBundleHash))
```

Comment:
seems unnecessary to check for the count if u call contain afterwards

Reply:
No it’s absolutely necessary because the contain is checking for absence. It’s a whitelist not a blacklist so if the list is empty (which it is by default) then it can be assumed that every bundle should be modified

Reply:
ah bruh yeah im stupid